sympy/calculus/util.py

```diff
@@ -253,6 +253,83 @@ def elm_domain(expr, intrvl):
         return _domain
 
 
+def periodicity(f, symbol):
+    """Returns the period of a given function, if possible.
+
```

Comment:
Can you write about Parameters, exceptions it raises and for which kind of functions(general form) it is working currently ?